tests/grid/grid_component.test.ts

```diff
@@ -1376,7 +1376,6 @@ describe("Copy paste keyboard shortcut", () => {
     const clipboardContent = "content" in clipboard ? clipboard.content : {};
     expect(clipboardContent).toMatchObject({
       "text/plain": "things",
-      "text/html": "things",
```

Comment:
Do we really have to remove those tests ? Can't you test for the string `<div> ....`